opencti-platform/opencti-graphql/src/utils/filtering.js

```diff
@@ -98,49 +102,51 @@ export const resolvedFiltersMapForUser = async (context, user, filters) => {
   return resolveUserMap;
 };
 
-export const convertFiltersFrontendFormat = async (context, user, filters) => {
-  // Grab all values that are internal_id that needs to be converted to standard_ids
-  const resolvedMap = await getEntitiesMapFromCache(context, SYSTEM_USER, ENTITY_TYPE_RESOLVED_FILTERS);
-  // Remap the format of specific keys
+const convertFiltersFrontendFormatContent = async (context, user, mainFilterGroup, resolvedMap) => {
```

Comment:
it's because we define filterGroups below... 
![image](https://github.com/OpenCTI-Platform/opencti/assets/75783086/77524dfc-56c9-4ffb-b27e-dc4de28781fb)